docs/topics/proc_deploying-the-configmap-booster-using-the-oc-cli-client.adoc

```diff
@@ -10,11 +10,15 @@
 
 .Prerequisites
 
-* The booster application created using the {launcher} application.
+ifndef::parameter-openshiftlocal[]
```

Comment:
Same as `proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc`.